lib/boot-module-files.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2020 Joyent, Inc.
+ */
+
+/*
+ * Read / write boot-time module files
+ */
+
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const assert = require('assert-plus');
+const mkdirp = require('mkdirp');
+const vasync = require('vasync');
+
+function generateBootModuleFile(opts, cb) {
+    assert.object(opts, 'opts');
+    assert.object(opts.log, 'opts.log');
```

Comment:
opts.log not used

Reply:
He, it was until I removed it after tests :-) I'll get rid of it.